app/bundles/NotificationBundle/Views/Popup/index.html.twig

```diff
@@ -111,7 +111,7 @@
             <p> Notifications are already enabled, you may close this window. </p>
 
             <p style="font-size: 12px">If you would like to unsubscribe from all notifications from
-                <span class="domainName"><?php echo $siteUrl; ?></span> click on the lock icon to the left of the address. </p>
+                <span class="domainName">{{ siteUrl }}</span> click on the lock icon to the left of the address. </p>
```

Comment:
The text here needs to be translated not added as is, I know it was added as text in the original code but it would be a great addition